### PR TITLE
In pom avoided unwanted copy files to webapp, added cross origin request support

### PR DESCRIPTION
**Pull request by RenukaGurumurthy**

### CODE REVIEW

**Comment by sheebans** [comment]
looks like your fork repository is not update to date.

**Comment by daniel1028** [comment]
yes..this is happened due to conflicts.It is resolved

File: insights-v1-api/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -23,6 +23,73 @@
...
+ -->        </filter>
+          <filter-mapping>
+        <filter-name>CORS</filter-name>
+        <servlet-name>dispatcher</servlet-name>
+        </filter-mapping>
+        <filter-mapping>
+        <filter-name>CORS</filter-name>
+        <url-pattern>/*</url-pattern>
+        </filter-mapping> 
   <servlet>
```
Comment (daniel1028) [inline insights-v1-api/src/main/webapp/WEB-INF/web.xml]: @RenukaGurumurthy 
Please remove atmosphere related tags

Comment (RenukaGurumurthy) [inline insights-v1-api/src/main/webapp/WEB-INF/web.xml]: This is in V1 module, which we are going to deprecate anyway.

File: insights-v3-api/pom.xml [hunk 2]
```diff
@@ -320,6 +310,23 @@
...
+		<dependency>
+		        <groupId>com.thetransactioncompany</groupId>
+		        <artifactId>cors-filter</artifactId>
+		        <version>1.8</version>
+		</dependency>
+		<!-- Added Dependency for CORS -->
+		 <dependency>
+		        <groupId>com.thetransactioncompany</groupId>
+		        <artifactId>java-property-utils</artifactId>
+		        <version>1.9.1</version>
```
Comment (sheebans) [inline insights-v3-api/pom.xml]: looks like this is duplicate entry for "java-property-utils" maven dependency 

@RenukaGurumurthy

**Pull request merged**